xtask/Cargo.toml

```diff
@@ -5,4 +5,22 @@ edition.workspace = true
 
 [dependencies]
 clap = { version = "4.5.4", features = ["derive", "env"] }
+color-eyre = "0.6.3"
+convert_case = "0.6.0"
+indexmap = { version = "2.2.6", features = ["serde"] }
+# onig = "6.4.0"
+phf_codegen = "0.11.2"
+regex = "1.10.4"
+serde = "1.0.200"
+serde_derive = "1.0.200"
+serde_ignored = "0.1.10"
+serde_json = "1.0.116"
+serde_path_to_error = "0.1.16"
+serde_with = "3.8.1"
+textwrap = { version = "0.16.1", default-features = false }
 thiserror = "1.0.58"
+tracing = "0.1.40"
+tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
+
+onig = { git = "https://github.com/rust-onig/rust-onig", revision = "fa90c0e97e90a056af89f183b23cd417b59ee6a2" }
```

Comment:
Why onig and not `regex` crate ?

Reply:
because I needed lookahead to correctly handle nested "color" tags in the regex. if you have up with an alternative in the Wiki help I'm all ears, `onig` is kinda broken atm, hence the git version and crates-io patch.